### PR TITLE
Pause messages - Fixes #296

### LINKED ISSUE #296: Addition of chat message for pause tag
Basic feature request:
- Addition of chat message for pause/unpause, there currently isn't really a proper notifications, just says PAUSED in green.
- Changing of [PAUSED] to red text instead of green.

### PR DESCRIPTION
**Pull request by Technoblazed**
Still needs testing but don't imagine it'll take too long... lol.

### CODE REVIEW

**Comment by Technoblazed** [comment]
@shavitush Completed push, tested & working.

File: translations/shavit-core.phrases.txt [hunk 1]
```diff
@@ -55,6 +55,16 @@
...
 	// ---------- Pauses ---------- //
+	"MessagePause"
+	{
+		"#format"	"{1:c},{2:c},{3:c}"
+		"en"		"{1}Timer has been {2}paused{3}."
+	}
+	"MessageUnpause"
+	{
+		"#format"	"{1:c},{2:c},{3:c}"
+		"en"		"{1}Timer has been {2}unpaused{3}."
```
Comment (shavitush) [inline translations/shavit-core.phrases.txt]: Maybe.. "resumed" instead?

Comment (Technoblazed) [inline translations/shavit-core.phrases.txt]: sure

**Comment by shavitush** [comment]
Thanks for contributing!

**Pull request merged**